tide_core.install

```diff
@@ -1687,3 +1687,21 @@ function tide_core_update_8072() {
     $role->save();
   }
 }
+
+/**
+ * Make Related link field mandatory and adds help text.
+ */
+function tide_core_update_8073() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [\Drupal::service('extension.list.module')->getPath('tide_core') . '/config/install'];
+  $config_id = 'field.field.paragraph.related_links.field_paragraph_link';
+  $link_config = \Drupal::configFactory()->getEditable($config_id);
+  $link_config_read = _tide_read_config($config_id, $config_location, FALSE);
+  $link_description = $link_config_read['description'];
+  $systems = $link_config_read['settings']['title'];
+  if ($link_config) {
+    $link_config->set('description', $link_description);
+    $link_config->set('settings.title', $systems);
+    $link_config->save();
+  }
```

Comment:
hey @edyuenyw 
The above looks too complicated. you probably just want to load the active  `field.field.paragraph.related_links.field_paragraph_link` and change the `description` and `title` directly.

Reply:
Thanks @vincent-gao,
Change made. Less is more 😄